packages/webamp/js/components/VerticalSlider.tsx

```diff
@@ -99,9 +101,16 @@ export default function VerticalSlider({
         }
         document.removeEventListener("mousemove", handleMouseMove);
         document.removeEventListener("mouseup", handleMouseUp);
+        if (type === "equalizer") {
+          sliderNode.removeEventListener("mouseleave", handleMouseUp);
+        }
       };
       document.addEventListener("mousemove", handleMouseMove);
       document.addEventListener("mouseup", handleMouseUp);
+      if (type === "equalizer") {
+        // If the mouse leaves the Slider area, trigger a MouseUp
+        sliderNode.addEventListener("mouseleave", handleMouseUp);
```

Comment:
Since this is now triggered on more than just mouse up, can we rename `handleMouseUp` to `handleRelease` or something like that?